sites/feedback/ajax/page-feedback-en.html

```diff
@@ -0,0 +1,74 @@
+<div class="gc-pft row" data-wb-jsonmanager='{
+	"name": "feedback",
+	"extractor": [
+		{ "interface": "locationHref", "path": "submissionPage" },
+		{ "selector": "title", "path": "pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "language" },
+		{ "selector": "meta[name=\"dcterms.type\"]", "attr": "content", "path": "themeopt" },
+		{ "selector": "meta[name=\"dcterms.subject\"]", "attr": "content", "path": "sectionopt" },
+		{ "selector": "meta[name=\"author\"]", "attr": "content", "path": "institutionopt" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">Page feedback</h3>
+			<form id="gc-pft-frm" action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk", "content":"#gc-pft-frm"}'>
+				<div data-wb-json='{
+					"url": "#[feedback]",
+					"mapping": [
+						{ "selector": "input", "attr": "name", "value": "/@id" },
+						{ "selector": "input", "attr": "value", "value": "/@value" }
+					]
+				}'>
+					<template>
+						<input type="hidden" name="" value="" />
+					</template>
+				</div>
+				<fieldset class="gc-pft-btns chkbxrdio-grp row row-no-gutters d-sm-flex flex-sm-wrap align-items-sm-center">
+					<legend class="col-xs-12 col-sm-7 nojs-col-sm-12 col-md-9 col-lg-8 text-center text-sm-left nojs-text-left mrgn-tp-sm pr-sm-3 nojs-pr-sm-0"><span class="field-name">Did you find what you were looking for?</span></legend>
+					<div class="col-xs-12 nojs-show">
+						<button type="submit" name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
+						<button type="submit" name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
```

Comment:
```suggestion
						<button name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
					</div>
					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
						<button name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
```

Just as a follow-up to https://github.com/wet-boew/GCWeb/pull/2098#discussion_r1104733082, it should become feasible to remove the yes buttons' ``type="submit"`` attributes once wet-boew/wet-boew#9534 lands.